app/src/ui/diff/text-diff.tsx

```diff
@@ -87,7 +84,7 @@ function highlightParametersEqual(
 ) {
   return (
     (newProps === prevProps || newProps.file.id === prevProps.file.id) &&
-    newState.document.diff.text === prevState.document.diff.text
+    newState.diff.text === prevState.diff.text
```

Comment:
Similarly to the `highlightParametersEqual` method in side-by-side-diff

```suggestion
    newState.diff.text === prevState.diff.text &&
    prevProps.fileContents?.file.id === newProps.fileContents?.file.id
```